src/views/domain-page/__fixtures__/domain-page-query-params.ts

```diff
@@ -1,9 +1,7 @@
 import { type PageQueryParamValues } from '@/hooks/use-page-query-params/use-page-query-params.types';
 import type domainPageQueryParamsConfig from '@/views/domain-page/config/domain-page-query-params.config';
 
-export const mockDomainPageQueryParamsValues: PageQueryParamValues<
-  typeof domainPageQueryParamsConfig
-> = {
+export const mockDomainPageQueryParamsValues = {
```

Comment:
Better split this per use case, to describe what is the actual needed params per scenario
e.g. mockDomainPageArchivalQueryParamsValues

Reply:
Good idea, but we'll need to refactor several tests for this. Can we discuss this on Thursday?